packages/app-store/fathom/components/EventTypeAppCardInterface.tsx

```diff
@@ -25,6 +25,7 @@ const EventTypeAppCard: EventTypeAppCardComponent = function EventTypeAppCard({
       }}
       switchChecked={enabled}>
       <TextField
+        className="mb-0 leading-4"
```

Comment:
@gitstart-calcom instead of adding this to every single app, can we maybe add this outside of the component so it applies to every app, also future apps?

Reply:
Sure, we are working on it 😄